### PR TITLE
update bench code

### LINKED ISSUE #2: bench code
@stjepang I didn't quite do the bench macro the way you said we should do it. Mostly because I forgot what it should be like. Could you maybe share again how it should be? Thanks!

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Closes #2. Also allows running setup code again inside the benchmark. Thanks!

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
Tests pass. Going to go ahead and merge this.

File: src/lib.rs [hunk 1]
```diff
@@ -143,22 +143,20 @@ pub fn test(_attr: TokenStream, item: TokenStream) -> TokenStream {
 ///
 /// ```ignore
```
Comment (taiki-e) [inline src/lib.rs:144]: test was ignored

Comment (yoshuawuyts) [reply src/lib.rs:144]: Ah dang. Yep definitely missed that.

Comment (taiki-e) [reply src/lib.rs:144]: Oh, doc tests run with `cargo build`, so it seems that we can't actually test `#[test]` and `#[bench]` via doc tests...

**Pull request merged**